### DISCUSSION TITLE
clang pgo support

### DISCUSSION THREAD

**Discussion by liu-song-6** [dynup/kpatch#1229] [General]
We are trying to build kernel with clang with PGO (Profile-Guided Optimization). Based on my limited understanding, kpatch-build doesn't support PGO at the moment. Is someone looking into this? If not, I will try to enable it. 

Thanks!

**Comment by sm00th** [comment]
I have no knowledge of anyone working on this so far, at least no one in Red Hat.

**Reply by liu-song-6** [reply]
Thanks for the confirmation. I will try to enable it.
